src/components/Navigation/Navbar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
```

Comment:
Kindly use the `NavLink` instead of `Link` for navigation. ⬇️
![image](https://user-images.githubusercontent.com/33268927/216074772-82cf6222-7a2c-4040-ad15-3fa6664e20f7.png)